### ISSUE TITLE
FlashMessage onDismiss button should have an aria-label

### ISSUE THREAD

**Issue by chawes13** [LaunchPadLab/lp-components#516]
![Image 2021-11-24 at 8 49 27 AM](https://user-images.githubusercontent.com/20426095/143250511-4a1bb238-93b5-4415-8aa5-12122c5c2d7a.jpg)
